src/spikeinterface/preprocessing/tests/test_filter.py

```diff
@@ -28,6 +161,8 @@ def test_filter():
     # other filtering types
     rec3 = filter(rec, band=500.0, btype="highpass", filter_mode="ba", filter_order=2)
     rec4 = notch_filter(rec, freq=3000, q=30, margin_ms=5.0)
+    rec5 = causal_filter(rec, direction="forward")
+    rec6 = causal_filter(rec, direction="backward")
```

Comment:
Since you created the new class, these two lines should go, right? Let me know if it is ok to eliminate them.

Reply:
No I would keep them ;)